spec/controllers/skills_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe SkillsController do
+    describe 'CRUD operaions' do
+      it 'index returns all skills ' do
+        get :index
+        skills = assigns(:skills)
+        expect(skills.sort).to eq skills().sort
+        expect(response).to render_template("index")
+      end
+
+      it 'post returns all skills ' do
+        title = 'new skill'
+        category_id = categories(:java).id
+        radar = "hold"
+        portfolio = "passiv"
+
+        post :create , params: { skill: { title: title, category_id: category_id, radar: radar, portfolio: portfolio} }
```

Comment:
Just pass the values straight into the creation call here instead of defining them on lines 13-16.

Reply:
but if i do it like this i can use the same vars to check if the value is actually the same